hack/create.sh

```diff
@@ -93,21 +93,26 @@ properties:
         clientId: $AZURE_AAD_CLIENT_ID
         secret: $AZURE_AAD_CLIENT_SECRET
         tenantId: $AZURE_TENANT_ID
+  networkProfile:
+    vnetCidr: 10.0.0.0/8
```

Comment:
How does this work? All network profile is 10.0.0.0-254 and masters is same as compute nodes and infra? how does IP address allocation work? Should it be it not be something like vnetCidr `10.0.0.0/23` and we give `10.0.0.0/24` to masters. `10.0.0.1/24` to infras, etc etc ?

Reply:
This matches what we were doing before, but it becomes user configurable so that the user can use a network range that fits with their peered vnet.  The precise details of what we use as defaults in create.sh are unimportant - and also do not change in this PR.  In the future we should probably hand out different ranges to master/infra/compute - that's #537.

Reply:
But does this works now if you hand the same subnet to each Pool? Just wondering